src/util/object_factory_parameters.cpp

```diff
@@ -74,6 +74,10 @@ void parse_object_factory_options(const cmdlinet &cmdline, optionst &options)
       "max-nondet-string-length",
       cmdline.get_value("max-nondet-string-length"));
   }
+  else if(!cmdline.isset("no-refine-strings"))
+  {
+    options.set_option("max-nondet-string-length", 10000);
```

Comment:
Why 10,000?  Does this affect performance?  If so is this a good choice?  Creating an array of 10,000 16-bit characters is a non-trivial amount for the SAT solver to handle.

Reply:
As documented here: https://github.com/diffblue/test-gen/blob/develop/doc/cl-options.md 
The performance should not be affected in any significant way. The value is anyway recommended to be set, so setting the default value is only to make sure that we do not trip over it if we forget.

Reply:
Has the option any effect if no-refine-strings is specified?

Reply:
No, the two options cannot be specified together: https://github.com/diffblue/cbmc/blob/88f2a96f6e105803760a85d6c725ef5165cbd48e/jbmc/src/jbmc/jbmc_parse_options.cpp#L281

Reply:
@svorenova : I don't have access to that documentation.

Reply:
@martin-cs Here is screenshot:
![image](https://user-images.githubusercontent.com/19727952/57535664-94028000-733a-11e9-8153-d0ed4d6648f4.png)

Reply:
That sounds like an argument for it to be 100 rather than 10000.

Reply:
I think that's because the doc is more concerned with test-generation than verification. If you are interested in the trace generated by jbmc it's better to use a reasonable value like 100, otherwise you can set it as high as you want.